### PR TITLE
Model-centric Setup Steps

### PR DESCRIPTION
**Pull request by evantahler**
This PR updates the Grouparoo Setup Steps to focus on our new Model-Centric UI.

Screen Shot 2022-01-07 at 4 20 43 PM

Notes:
* The step to rename your cluster remains
* This PR updates the CTAs to use the new `/model/{name}/overview` page

### Development

- [x] Application changes have been tested appropriately

### Impact

- [x] Code follows company security practices and guidelines
- [x] Security impact of change has been considered
- [x] Performance impact of change has been considered
- [x] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:

### Code review

- [x] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [x] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

### CODE REVIEW

File: core/src/modules/ops/setupSteps.ts [hunk 1]
```diff
@@ -1,272 +1,128 @@
...
+      runModes: ["cli:start", "cli:run", "cli:config"],
+    },
+    {
+      key: "configure_a_model",
+      title: "Define your Model Data",
+      description:
+        "Create Sources and Properties to define the data in your Grouparoo Model.",
+      href: modelId ? `/model/${modelId}/overview` : "/models",
+      cta: "Configure the Model",
+      helpLink: `${configURL}/models`,
```
Comment (teallarson) [inline core/src/modules/ops/setupSteps.ts]: Would `/sources` make more sense here?  The `/models` page doesn't seem like a great match.  

I just put a story in for docs changes in general (editing the first step in each of the Config docs to reference the model overview page, updating screenshots)

File: core/__tests__/actions/setupSteps.ts [hunk 2]
```diff
@@ -118,35 +118,28 @@ describe("actions/setupSteps", () => {
...
-      async (topic) => {
-        connection.params = { csrfToken, modelId: null };
-        const { setupSteps: setupStepsNoModelId } =
-          await specHelper.runAction<SetupStepsList>(
-            "setupSteps:list",
-            connection
-          );
-        let destinationStep = setupStepsNoModelId.find(
-          (s) => s.key === `create_a_${topic}`
+    test("destination setupSteps href changes based on modelId", async () => {
```
Comment (teallarson) [inline core/__tests__/actions/setupSteps.ts:149]: Wouldn't the "Configure Your Model" step also be by model id?

**Review by teallarson — APPROVED** [review]
Nice!  A couple questions I left as comments, but overall looks good.

File: core/src/modules/ops/setupSteps.ts (hunk 1 above)
Comment (evantahler) [reply core/src/modules/ops/setupSteps.ts]: I think the `/{model}/sources` page is going away soon, so I didn't want to use it - https://www.pivotaltracker.com/story/show/180237163

File: core/__tests__/actions/setupSteps.ts (hunk 2 above)
Comment (evantahler) [reply core/__tests__/actions/setupSteps.ts:149]: Test added in https://github.com/grouparoo/grouparoo/pull/2773/commits/5f9a6541528e5b65ae9cf55f128b8250fb0cc38a

File: core/src/modules/ops/setupSteps.ts (hunk 1 above)
Comment (teallarson) [reply core/src/modules/ops/setupSteps.ts]: Sorry, I was not clear!  I meant the docs page for configuring sources as the help link.

Comment (evantahler) [reply core/src/modules/ops/setupSteps.ts]: Ah, yes! I'll add a note to the new story to confirm these URLs as well once the new docs are up

**Pull request merged**